### PR TITLE
Unit Tests for Engine/aim_repo

### PR DESCRIPTION
**Pull request by jialin-wu-02**
- Enable unit tests for core functionality within `engine/aim_repo`

There is something strange about aim_profile: `aimprofile.init()` keeps failing. Thus only include the tests for `aim_repo`.

### CODE REVIEW

File: tests/engine/test_aim_repo.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import unittest
+import tempfile
+import os
+
+
+class TestAimRepo(unittest.TestCase):
+    @staticmethod
+    def get_aim_repo(directory):
+        """
+        Searches for .aim repository in the givne directory
```
Comment (gorarakelyan) [inline tests/engine/test_aim_repo.py:12]: Maybe `given`?

**Comment by gorarakelyan** [comment]
I would also add `pytest.ini` file for convenience. Like this:
```
[pytest]
python_paths = tests
addopts = --pdbcls=IPython.terminal.debugger:Pdb
```

**Comment by jialin-wu-02** [comment]
> I would also add `pytest.ini` file for convenience. Like this:
> 
> ```
> [pytest]
> python_paths = tests
> addopts = --pdbcls=IPython.terminal.debugger:Pdb
> ```

Thanks! Wondering do I put the file in `tests/engine` or `tests`?

**Comment by gorarakelyan** [comment]
In main directory, from where command `pytest` is going to be executed

File: tests/engine/test_aim_repo.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            })
+            # should be two files (config, test_file_two)
+            self.assertEqual(len(ar.ls_commit_files('test', '2')), 2)
+
+
+
+if __name__ == '__main__':
+    unittest.main()
+
+
```
Comment (gorarakelyan) [inline tests/engine/test_aim_repo.py:139]: PEP8 requires only one blank line at the end of a file

**Comment by gorarakelyan** [comment]
Have you enabled PEP8 style guide checking in your IDE/code editor?

**Pull request merged**